test_scripts/API/AutoCompleteList/016_ATF_SetGlobalProperties_N_AutoCompleteList_HMI_not_Respond_GENERIC_ERROR.lua

```diff
@@ -0,0 +1,59 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL SDL respond <success = false, resultCode = "GENERIC_ERROR"> to mobile app
+--
+-- Description:
+-- SDL must: tranfer SetGlobalProperties_request with <autoCompleteList> param param to HMI
+-- respond with <resultCode_received_from _HMI> to mobile app
+--
+-- Performed steps:
+-- 1. Register Application.
+-- 2. Mobile send RPC SetGlobalProperties with <autoCompleteList> 
+-- 3. HMI does NOT respond during <DefaultTimeout>
+-- 4. SDL respond <success = false, resultCode = "GENERIC_ERROR"> to mobile app
+---------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+
+--[[ General Precondition before ATF start ]]
+config.defaultProtocolVersion = 2
+commonSteps:DeleteLogsFileAndPolicyTable()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function  Test:SetGlobalProperties_WithNotResponse_from_HMI()
+  --mobile side send SetGlobalProperties request
+  local cid = self.mobileSession:SendRPC("SetGlobalProperties",
+     {
+      keyboardProperties =
+      {
+        keyboardLayout = "QWERTY",
+        keypressMode = "SINGLE_KEYPRESS",
+        limitedCharacterList =
+        {
+          "a"
+        },
+        language = "EN-US",
+        autoCompleteText = "Text_1, Text_2",
+        autoCompleteList = {"List_1, List_2", "List_1, List_2"}
+      }
+    })
+--hmi side: Default timeout, SDL recieved GENERIC_ERROR
```

Comment:
@Aleksandriys, in test is missed step for ensuring that HMI sends nothing, you just expect GENERIC_ERROR to mobile(comment "--hmi side: Default timeout, SDL recieved GENERIC_ERROR" is not correct).
Please add check, that hmi sends nothing